### PR TITLE
add dashboard diagnostics docs

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Add Dashboard diagnostics docs.

This PR must be merged after https://github.com/pingcap/docs/pull/2655.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3273 https://github.com/pingcap/docs-cn/pull/3247  https://github.com/pingcap/docs-cn/pull/3366
- Other reference link(s):

### CODE REVIEW

File: dashboard/dashboard-diagnostics-access.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: TiDB Dashboard Cluster Diagnostics Page
+summary: Learn how to use the cluster diagnostics.
```
Comment (TomShawn) [inline dashboard/dashboard-diagnostics-access.md:3]:
```suggestion
summary: Learn how to use the cluster diagnostics page.
```

File: dashboard/dashboard-diagnostics-access.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+2. Set the range duration. Generally, this duration is the duration of system anomalies, such as 5 minutes.
+3. Enable **Compare by baseline**.
+4. Set the baseline start time, which is the start time of the range (to compare) in which the system is normal, such as `2020-05-21 14:30:00`.
+5. **Click Start**.
+
+![Generate comparison report](/media/dashboard/dashboard-diagnostics-gen-compare-report.png)
+
+Then wait for the report to be generated and click **View Full Report**.
+
+In addition, the generated diagnostic report is displayed in the list on the main page of the diagnostic report. You can click to view the previously generated reports which do not need to be generated for a second time.
```
Comment (crazycs520) [inline dashboard/dashboard-diagnostics-access.md:62]:
```suggestion
In addition, the historical generated diagnostic report is displayed in the list on the main page of the diagnostic report. You can click to view the previously generated reports which do not need to be generated for a second time.
```

File: dashboard/dashboard-diagnostics-report.md [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+
+![Sample report](/media/dashboard/dashboard-diagnostics-example-table.png)
+
+In the image above, **Total Time Consume** in the top blue box is the report name. The information in the red box below explains the meaning of this report and the meaning of each field in the report.
+
+In this report, some small buttons are described as follows:
+
+* **i** icon: You can move your mouse to the **i** icon to see the explanatory note of the row.
+* **expand**: Click **expand** to see more detailed information of this monitoring metric. For example, the detailed information of `tidb_get_token` in the image above includes the monitoring information of each TiDB instance's latency.
+* **fold**: Contrary to **expand**, the button is used to fold detailed monitoring information.
```
Comment (toutdesuite) [inline dashboard/dashboard-diagnostics-report.md:32]: 实际生成的诊断报告中是 collapse

File: dashboard/dashboard-diagnostics-access.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+The steps above generate a diagnostic report for the time range from `2020-05-21 14:40:00` to `2020-05-21 14:50:00`. After clicking **Start**, you can see the interface below. **Progress** is the progress bar for generating the report. After the report is generated, click **View Full Report**.
+
+![Report progress](/media/dashboard/dashboard-diagnostics-gen-process.png)
+
+## Generate comparison diagnostic report
+
+If the system is abnormal at a certain point, such as QPS jitter or the latency becomes higher, a diagnostic report can be generated that compares the system in the abnormal time range with the system in the normal time range. For example:
+
+* Abnormal time range: `2020-05-21 14:40:00` ~ `2020-05-21 14:45:00`. Within this time range, the system is abnormal.
+* Normal time range: `2020-05-21 14:30:00` ~ `2020-05-21 14:35:00`. Within this time range, the system is normal.
```
Comment (TomShawn) [inline dashboard/dashboard-diagnostics-access.md:48]:
```suggestion
* Normal time range: `2020-05-21 14:30:00` - `2020-05-21 14:35:00`. Within this time range, the system is normal.
```

File: dashboard/dashboard-diagnostics-report.md (hunk 3 above)
Comment (TomShawn) [inline dashboard/dashboard-diagnostics-report.md:32]:
```suggestion
* **collapse**: Contrary to **expand**, the button is used to fold detailed monitoring information.
```

File: dashboard/dashboard-diagnostics-report.md [hunk 5]
```diff
@@ -0,0 +1,369 @@
...
+
+In addition, the `TOTAL_TIME` and `TOTAL_COUNT` metrics in this report are monitoring data read from Prometheus, so calculation inaccuracy might exits in their statistics.
+
+Each part of this report is introduced as follows.
+
+### Basic information
+
+#### Report Time Range
+
+The time range for generating the report includes the start time and end time.
```
Comment (TomShawn) [inline dashboard/dashboard-diagnostics-report.md:44]:
```suggestion
The time range for generating the diagnostics report includes the start time and end time.
```

File: dashboard/dashboard-diagnostics-access.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
+---
+title: TiDB Dashboard Cluster Diagnostic Page
+summary: Learn how to use the cluster diagnostic page.
+category: how-to
+---
+
+# TiDB Dashboard Cluster Diagnostics Page
```
[the remark this answers is not in the document]
Comment (TomShawn) [reply dashboard/dashboard-diagnostics-access.md:7]: But in the actual UI, we can see **Cluster Diagnostics**.
image

[the remark this answers is not in the document]
Comment (yikeke) [reply dashboard/dashboard-diagnostics-access.md:7]: Diagnostics is correct @toutdesuite

**Pull request merged**